lib/connection/connection_config.js

```diff
@@ -147,7 +147,7 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
   {
     // username is not required for oauth authenticator
     if (!Util.exists(options.authenticator) ||
-      options.authenticator !== authenticationTypes.OAUTH_AUTHENTICATOR) 
+      options.authenticator !== authenticationTypes.OAUTH_AUTHENTICATOR )
```

Comment:
if you change formatting then fix also `{}` position but maybe you shouldn't in this PR in this place